### PR TITLE
Theme improvements

### PR DESCRIPTION
**Pull request by Holzhaus**
- Clean up `_templates` directory
- Add download link for PDF version
- Override hyperlink text color for toctree

### CODE REVIEW

File: source/_static/css/widget-sidebar.css [hunk 1]
```diff
@@ -0,0 +1,143 @@
+.rst-widget-sidebar::-webkit-scrollbar {
+  /* This is the magic bit */
```
Comment (rryan) [inline source/_static/css/widget-sidebar.css]: Did you write this? If so could you write a more descriptive comment for future archaeologists? :)

File: source/_static/js/widget-sidebar.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+(function() {
```
Comment (rryan) [inline source/_static/js/widget-sidebar.js]: Curious if you wrote all this (the custom sidebar widget js/css) or whether it came from a ready-made gist or something someone else made? If the latter, can we include a link to the source so we can keep it up to date and give credit?

Comment (Holzhaus) [reply source/_static/js/widget-sidebar.js]: Nope, I wrote the JS bits myself.

File: source/_static/css/widget-sidebar.css (hunk 1 above)
Comment (Holzhaus) [reply source/_static/css/widget-sidebar.css]: I used the css declarations from the RTD theme and mass replaced `rst-versions` with `rst-widget-sidebar`. The line you referenced was copied from from StackOverflow to hide scrollbars in webkit/webengine browsers. I can add the link.

**Comment by ywwg** [comment]
how long after this is submitted will the site be updated? (automatic / instant?)

**Comment by rryan** [comment]
> how long after this is submitted will the site be updated? (automatic / instant?)

This Jenkins job is in charge of publishing: https://builds.renegadetech.mixxx.org/job/v2.2/job/manual-2.2-publish/

Instant is a stretch (building in all languages and uploading to hostgator is strangely slow) :) but it's automatic.

**Comment by ywwg** [comment]
sounds good, thanks

**Comment by Holzhaus** [comment]
Merge?

**Comment by ywwg** [comment]
rryan approved so I'll push the button

**Pull request merged**